indico/modules/attachments/models/folders.py

```diff
@@ -50,7 +50,7 @@ def __auto_table_args():
         return (db.CheckConstraint(default_inheriting, 'default_inheriting'),
                 db.CheckConstraint('is_default = (title IS NULL)', 'default_or_title'),
                 db.CheckConstraint('not (is_default and is_deleted)', 'default_not_deleted'),
-                {'schema': 'attachments'})
```

Comment:
what happened to this?!

Reply:
Lost due to mis-reading braces..., readded